app/src/main/java/com/android/calendar/AllInOneActivity.java

```diff
@@ -267,12 +267,12 @@ protected void onCreate(Bundle icicle) {
         // This needs to be created before setContentView
         mController = CalendarController.getInstance(this);
 
-        // Create notification channel
-        AlertService.createChannels(this);
-
         // Check and ask for most needed permissions
         checkAppPermissions();
 
+        // Create notification channels
+        AlertService.createChannels(this);
```

Comment:
Moving this after the permissions check, because the notification channel creation now needs calendar permission or it doesn't run